modal-js/test/legacy/cloud_bucket_mount.test.ts

```diff
@@ -0,0 +1,130 @@
+import { CloudBucketMount, Secret } from "modal";
```

Comment:
All the tests under `modal-js/test/legacy/` are copies of the corresponding tests in `modal-js/test/`, but using the now legacy APIs.